app/code/Magento/Catalog/Block/Product/ListProduct.php

```diff
@@ -40,17 +44,17 @@ class ListProduct extends AbstractProduct implements IdentityInterface
     /**
      * Catalog layer
      *
-     * @var \Magento\Catalog\Model\Layer
+     * @var Layer
```

Comment:
Undefined class Layer, use FQCN.

Reply:
Disagree, why not add a proper import instead?

Reply:
Or add proper import:)

Reply:
Solved. Thank you for your review.